### ISSUE TITLE
MS_Celeb_1M

### ISSUE THREAD

**Issue by lyuchuny3** [davidsandberg/facenet#179]
Hi, I have tested your pretrained model with MS_Celeb_1M dataset on LFW, the results are pretty good:

average precsion(ap):  0.999541130076
best_threshold :0.328682, accuracy :0.992833
auc 0.999439444444 	

I would like to train it by myself with MS_Celeb_1M, but this dataset are not clean and with lots of noise. would you please send me the cleaned_list for training of this dataset(MS_Celeb_1M) to my mail : lyuchuny3@foxmail.com. Thanks very much ^_^

**Comment by hardfish82** [comment]
I need too.:) Thanks.
hardfish82@gmail.com

**Comment by davidsandberg** [comment]
The plan is to write a wiki page about how to generate the "clean list" and use it for training, but I'm currently quite busy and haven't had time to do that.

**Comment by mhaghighat** [comment]
@lyuchuny3 : 
Is this _best_threshold_ you've measured for the 128-dimensional model or the 1792-dimensional one?

**Comment by SeanGuoXuan** [comment]
I have the same issue on cleaning MS_Celeb_1M dataset. What would be the best approach to clean the noisy data? @davidsandberg appreciate if you can give some hints on this task? 

Maybe, we should use pre-trained model to evaluate images in each folder, generate 128-d value for each picture, if one image has distance to the mean value beyond threshold shall be dropped.

**Comment by ugtony** [comment]
@SeanGuoXuan 
What you suggested is exactly what davidsandberg did to get his uploaded MsCeleb1M model. 
That could clean the dataset to some extent, but the difficult and correct examples(which are important) and the incorrect ones are still inseparable. So it is difficult to set a threshold.

**Comment by yekeping** [comment]
@ugtony  would you please tell me if I change the validation dataset,what parameters should I change beside exchanging the file folder of LFW with mine!BTW,what dataset do you use for training and what is the performance?I use the CASIA,but not clean.the accuracy on my dataset is about 85%
